ontotext-yasgui-web-component/src/models/keyboard-shortcut-description.ts

```diff
@@ -31,3 +34,5 @@ export enum EXPLAIN_PLAN_TYPE {
   EXPLAIN= 'explain',
   CHAT_GPT_EXPLAIN = 'gpt'
 }
+
+export type CreateKeyboardShortCutFunction = () => KeyboardShortcutDescription;
```

Comment:
CreateKeyboardShort`Cut`Function should be `CreateKeyboardShort`cut`Function`

Reply:
Fixed